### PR TITLE
Update  ..sequence-with-broker-trigger/README.md

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes 
Remove duplicate statement in the wrong location
Reword incorrect statement

It makes sense to cherry-pick this to the release branch as well.

### CODE REVIEW

File: docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md [hunk 1]
```diff
@@ -154,12 +154,8 @@ spec:
...
 ```
 
-Here, if you are using different type of Channel, you need to change the
-spec.channelTemplate to point to your desired Channel. Also, change the
-spec.reply.name to point to your `Broker`
-
-Change `default` below to create the `Sequence` in the Namespace where you have
-configured your `Broker`.
+Change `default` below to create the `PingSource` in the Namespace where you have
+configured your `Broker` & `Sequence`.
```
Comment (abrennan89) [inline docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md:158]:
```suggestion
Change the `default` namespace below to create the PingSource in the namespace where you have
configured your broker and sequence.
```

File: docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md [hunk 2]
```diff
@@ -184,7 +180,7 @@ spec:
       name: sequence
 ```
 
-Change `default` below to create the `Sequence` in the Namespace where you have
+Change `default` below to create the `Trigger` in the Namespace where you have
```
Comment (abrennan89) [inline docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md:183]:
```suggestion
Change `default` namespace below to create the trigger in the namespace where you have
```

File: docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md [hunk 3]
```diff
@@ -239,7 +235,7 @@ pods.
 kubectl -n default get pods
 ```
 
-Then look at the logs for the event-display pod:
+Then look at the logs for the sequence-display pod:
```
Comment (abrennan89) [inline docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md:238]:
```suggestion
Look at the logs for the `sequence-display` pod:
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Some minor comments :)

**Comment by abrennan89** [comment]
@xtreme-sameer-vohra I think you need to push again if you accepted comments because the CLA is weird, sorry! That should fix it though afaik.

**Review by abrennan89 — APPROVED** [review]
/lgtm
/hold

@knative/eventing-wg-leads can someone OK this from a technical perspective and remove the hold when you're ready? Thanks.

**Comment by xtreme-sameer-vohra** [comment]
Hey @abrennan89 
I just went through the README.md and added a commit to make the language consistent across the page

re: the google cla, I wonder 🤔  if it can't properly handle the shared commits when I accepted your suggestions. In either case, I just pushed again

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**